webapp/package.json

```diff
@@ -58,7 +58,7 @@
     "sass": "1.60.0",
     "sass-loader": "10.4.1",
     "style-loader": "1.2.1",
-    "typescript": "4.0.3",
+    "typescript": "4.2.3",
```

Comment:
This is adding a lot to the diff of the PR

Reply:
@mickmister Without these changes the `ci-lint` will fail. Let me know if you want us to find any other way around to fix the failing lint test other than this.